Calendars/Calendars.Plugin/CrossCalendars.cs

```diff
@@ -31,7 +31,7 @@ public static ICalendars Current
 
     static ICalendars? CreateCalendars()
     {
-#if NETSTANDARD1_0
+#if NETSTANDARD1_0 || (NET6_0 && !ANDROID && !IOS && !MACCATALYST && !WINDOWS)
```

Comment:
nit: You can use just `NET` here. It's only defined in `net5` and `net6` 
```suggestion
#if NETSTANDARD1_0 || (NET && !ANDROID && !IOS && !MACCATALYST && !WINDOWS)
```